### DISCUSSION TITLE
你知道cold stone 可以免費加料嗎

### DISCUSSION THREAD

**Discussion by yayun601506** [1101-GS4719-C/Python-Community-GS4719-C#2011] [閒聊八卦]
之前在Threads 上看到人家分享cold stone可以免費加料!
這麼好康的是肯定要衝了
上禮拜跟我朋友去sogo吃cold stone實測過了
是真的!
如果點的冰裡面有三種料就可以三種都各加一份

**Comment by jeff110303549** [comment]
這個資訊真的蠻實用，個人相對與其他連鎖的冰淇淋店像31、Hagen-Dazs 等比起來更喜歡cold stone 多樣的變化，不過有時候也會覺得有些品項甜到有負擔，可能還是得要好好克制吧。另外，我記得中國信託的優惠卷裡面也有一個是第二件半價的，推薦給其他喜歡享受他們家冰淇淋的同學們

**Comment by 155Julia** [comment]
我完全不知道竟然有這種好康，非常感謝版主的無私分享，我認為好的東西跟大家分享真的是一種很棒的美德，再次感謝，下次吃cold stone我會嘗試看看免費加料的！

**Comment by yubo0oo** [comment]
之前在Threads上看到有人分享Cold Stone可以免費加料，我覺得這真的是太棒了，絕對要去試試看！上週我跟朋友去了SOGO的Cold Stone實測了一下，結果發現這是真的！

當我們點了冰淇淋後，裡面有三種料，他們真的讓我們每種料都各加了一份，完全免費。我覺得這個活動實在太划算了，Cold Stone的冰淇淋本來就很好吃，加上這麼多料更是美味無比。

如果你也喜歡Cold Stone，我覺得一定要去試試這個免費加料的活動。趕快約上朋友，一起去享受這個美味又超值的冰淇淋體驗吧！

**Comment by Tchang323** [comment]
好酷喔! 下次去吃，話說好久沒吃coldstone的冰淇淋了
